docs/docs/UserGuide/configuration.md

```diff
@@ -0,0 +1,58 @@
+# BigDL Configuration
+BigDL uses Java properties to control its behavior. Here's the list of
+these properties.
+
+## Logging
+* bigdl.utils.LoggerFilter.disable
+
+Default value is **false**. Disable redirecting logs of Spark and BigDL to a file.
+
+* bigdl.utils.LoggerFilter.logFile
+
+Default value is **Current_Working_Directory/bigdl.log**. Where is the redirecting log.
+
+* bigdl.utils.LoggerFilter.enableSparkLog
+
+Default value is **true**. Enable redirecting Spark logs to logFile
+
+## Mode
+* bigdl.localMode
+
+Default value is **false**. Whether BigDL is running on a local JVM.
+
+* bigdl.engineType
+
+Default value is **mklblas**. Specify which kind of native library to speed up the computing.
```

Comment:
Looks like currently we only support mklblas. Do we need clarify that or remove this config here?